scripts/sysman/memory.yml

```diff
@@ -132,6 +132,21 @@ members:
       desc: "[out] The total allocatable memory in bytes (can be less than the `physicalSize` member of $s_mem_properties_t)"
 --- #--------------------------------------------------------------------------
 type: struct
+desc: "Extension properties for Memory State"
```

Comment:
Please share if we would be adding Enable / Disable of page offlining feature in a new API ?

Reply:
ECC has to be disabled to disable this feature

Reply:
Yes, ECC disable will disable this feature